kura/src/lib.rs

```diff
@@ -0,0 +1,131 @@
+/// Main entity in this crate is `Kura`.
+/// You should start usage of `Kura` via initialization.
+/// For example you can initialize `Kura` with full set of validations:
+/// ```
+/// use kura::Kura;
+///
+/// let kura = Kura::strict_init();
+/// ```
+
+/// High level data storage representation.
+/// Provides all necessary methods to read and write data, hides implementation details.
+struct Kura {}
+
+impl Kura {
+    /// Kura reads all transactions in all block keeping its order without any validation.
+    /// Better to use only for operations with no expectations about correctnes.
+    fn fast_init() -> Self {
+        Kura {}
+    }
+
+    /// `Kura::fast_init` with transactions and blocks validation (signatures correctness and business rules).
+    fn strict_init() -> Self {
+        Kura::fast_init()
```

Comment:
Looks good. For the MVP I would implement fast_init first and then strict_init later on, once we have the MVP.